package.json

```diff
@@ -13,15 +13,18 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "lint": "eslint --ext .js,.jsx src/client",
-    "lint:fix": "eslint --ext .js,.jsx src/client --fix"
+    "lint:fix": "eslint --ext .js,.jsx src/client --fix",
+    "style:lint": "stylelint src/client/assets/*.{css,scss}",
```

Comment:
I'm just curious, you've defined which files you'd like to run the lint command in `lint-staged`.
Do you still need to specify target files in `style:lint` and `lint` ?

Btw, I think maybe it's good to rename the `lint` (for example, `script:lint`) because `lint` sounds too general to only run `eslint` 🤔

Reply:
Hmm, I guess this part I need to do more research and get more understanding. I think I got an error message if I didn't specify the target files but not sure. Btw, thanks for the suggestion:)